lib/dependencies/HarmonyModulesPlugin.js

```diff
@@ -103,6 +104,9 @@ class HarmonyModulesPlugin {
 						return;
 
 					new HarmonyDetectionParserPlugin(this.options).apply(parser);
+					new HarmonyNode12CreateRequireDependencyParserPlugin(
```

Comment:
Not only related to node v12, should be renamed

Reply:
`createRequire` exported from `module` firstly comes from Node 12 API, I'm following the TypeScript naming (they choose to name those codegen targets as `Node12` and `NodeNext`)

Reply:
We don't have to follow the name in ts, I don't think we will have more `createRequire` dependencies